src/components/memorygame/MemoryGameFlippableCard/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+// check https://github.com/AaronCCWong/react-card-flip for 'react-card-flip' details
+import ReactCardFlip from "react-card-flip";
+import { Card } from "antd";
+import questionMark from "../../../images/memoryGameFrontImage.png";
+import "./index.css";
+
+// targetState example
+// {
+//   cardKey: "alternateEnergy0",
```

Comment:
now that this comment has been copied in multiple places that would not be acceptable as committed code, because it means that if i add a field to this state i need to add it to at least 2+ places (i think this is the same as the other comment in the other PR right?)

what you probably want to do is put a comment at the top of the main memory game component that says this. here, all you need to write is "see MemoryGameBoard/index.js for an example of what this object looks like" (it's a 'pointer' to that documentation)

Reply:
PTAL